### PR TITLE
Every good piece of software needs a figlet splash!

### PR DESCRIPTION
**Pull request by Ichbinjoe**
this pr is dope so just press merge

### CODE REVIEW

File: proxy/src/main/java/com/velocitypowered/proxy/Velocity.java [hunk 1]
```diff
@@ -20,6 +20,13 @@
     public static void main(String... args) {
         startTime = System.currentTimeMillis();
         logger.info("Booting up Velocity {}...", Velocity.class.getPackage().getImplementationVersion());
+        logger.info("");
+        logger.info("           _            _ _         \n" +
+                    "__   _____| | ___   ___(_) |_ _   _ \n" +
+                    "\\ \\ / / _ \\ |/ _ \\ / __| | __| | | |\n" +
+                    " \\ V /  __/ | (_) | (__| | |_| |_| |\n" +
+                    "  \\_/ \\___|_|\\___/ \\___|_|\\__|\\__, |\n" +
+                    "                              |___/");
```
Comment (astei) [inline proxy/src/main/java/com/velocitypowered/proxy/Velocity.java]: Doesn't have enough 🏃 - you can do better!

Comment (connorhartley) [reply proxy/src/main/java/com/velocitypowered/proxy/Velocity.java]: More. MORE!

Comment (Ichbinjoe) [reply proxy/src/main/java/com/velocitypowered/proxy/Velocity.java]: Added more velocity

**Comment by phase** [comment]
This think this PR is very vital to furthering the Minecraft ecosystem and will improve Velocimeme 10-fold.

**Comment by astei** [comment]
On a very serious note, this isn't really needed. I'm willing to tolerate a degree of silliness but I can see this going out of control.

**Pull request closed without merging**